.ruby-version

```diff
@@ -1 +1 @@
-3.0.2
\ No newline at end of file
+3.0.3
```

Comment:
I'd revert this and ship the update as part of a different PR.

Reply:
- I don't know why but trying to install `3.0.2` using `rbenv` was impossible for me. `3.0.3` didn't gave me any troubles, that's why I updated it, as it wasn't possible for me to use `3.0.2`.
- I will update that same command on `tuist build`.
- If you don't find this flag useful feel free to reject it 👍

Reply:
I'll open a PR with the update. Once merged on master rebase and we are good.

Reply:
https://github.com/tuist/tuist/pull/3817